### PR TITLE
feat: Colored INOP labels based on github branch

### PR DESCRIPTION
**Pull request by frankkopp**
### Colored Label

Adds copying of DDS files into the aircraft folder based on the current branch to the build.js.

This added two DDS files to the `src/Textures/decals 4k` folder 
- one Stable and Development: A320NEO_COCKPIT_DECALSTEXT_ALBD.TIF-yellow.dds
- one for all other branches: A320NEO_COCKPIT_DECALSTEXT_ALBD.TIF-red.dds

This will help recognize the version user uses by simply looking at the screenshot. This will be very helpful for the support team. 

The effect in the cockpit is: 
- No Sticker = Stable
- Yellow = Development
- Red = Experimental and other dev branches

Side note: Once WX and radar are operational again we should move the sticker and add another text. 

### Livery folder rename

This PR also renames the default liveries which has the effect that they are now sorted first on most users' systems and therefore also become default after switch aircraft.
It also fixes the meta data in the Aircraft Selection in development mode showing FlyByWire as the creator instead any other first livery's meta data. 

The change only adds a single "_" to the folder name so it if users have a livery also with a folder name starting with "_" it might be sorted before our default livery. 

## Screenshots (if necessary)
INOP Label:
image
image

Livery before:
image

Livery after:
image
image
image

## References
N/A

## Additional context
One of the most common questions in the Discord #support channel is which version is used and to provide a screenshot. 
With this change it is immediately apparent which version the users uses and if it is supported.

Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions

- Make sure the INOP label is red with white font. 
  - Stable and Development can't be tested during PR QA - needs a developer to test the code. 

- Check if default livery is sorted first in the livery list. If not check the folder name of the livery sorted first and confirm if it is alphabetically sorted before "_FlyByWire_A320_NEO-LIVERY"

Check if you can find any issues with selecting liveries and also start a flight if the default liveries are working. 
Should not have any impact on flight but can't hurt do a flight with this PR.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: scripts/build.js [hunk 1]
```diff
@@ -82,3 +82,38 @@ fs.writeFileSync(path.join(A32NX, 'manifest.json'), JSON.stringify({
...
+    const YELLOW_DDS = '/Textures/decals 4k/A320NEO_COCKPIT_DECALSTEXT_ALBD.TIF-yellow.dds';
+    // destination will be created or overwritten by default.
+    fs.copyFile(path.join(SRC_FOLDER, YELLOW_DDS), path.join(A32NX, TARGET_DSS),
+        (err) => {
+            if (err) {
+                throw err;
+            }
+            console.log('copying ' + YELLOW_DDS + ` TO ` + TARGET_DSS + "failed: " + err);
+        });
+}
```
Comment (aguther) [inline scripts/build.js:119]: I recommend doing the following:

```
if stable
  -> log that stable detected and no action needed
else if master
  -> yellow
else
  -> red
end
```

**Review by aguther — CHANGES_REQUESTED** [review]

File: scripts/build.js (hunk 1 above)
Comment (frankkopp) [reply scripts/build.js]: A bit shorter and assuming default DDS for Stable is in aircraft folder. 
```
if (GIT_BRANCH === 'master') {
 -> yellow 
} else if (edition !== 'stable') {
 -> red
}
```

**Review by aguther — APPROVED** [review]

**Comment by D4rknessCodez** [comment]
Quality Assurance Trainee Report

Discord: Darkness🍁#7734
Object of testing: #6467
Tier of Testing: 1
Date: (03/01/2022)

Testing Process: 
- Checked the liveries tab before loading, the default livery was sorted first followed by the Christmas livery and airbus livery
- Loaded into the sim, the INOP tags were visible in both applied areas.

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
Everything went well, i like the addition of the INOP stickers. Nice work!

**Comment by ShadowFrost3** [comment]
Quality Assurance Tester Report


Object of testing: #6467
Tier of Testing:1
Date: 05/01/2022

Testing Process: 
Livery order - checked
Grammar errors - checked
INOP stickers red - checked 

Testing Results: Passed

**Comment by ErikNL80** [comment]
No merge in the experimental version?

**Comment by frankkopp** [comment]
> Dev

This should happen automatically as we regularly update Exp from Dev.

**Comment by igor8518** [comment]
Good day, please tell me how these dds were obtained, and how can they be opened? Thanks!

**Pull request merged**